src/Mod/PartDesign/App/FeatureChamfer.cpp

```diff
@@ -76,6 +76,9 @@ Chamfer::Chamfer()
     Angle.setConstraints(&floatAngle);
 
     ADD_PROPERTY_TYPE(FlipDirection, (false), "Chamfer", App::Prop_None, "Flip direction");
+    ADD_PROPERTY_TYPE(UseAllEdges, (false), "Base", App::Prop_None,
+             "Chamfer all edges if true, else use only those edges in Base property\n"

```

Comment:
There should be a full stop before the newline here.

Reply:
What do you mean?  A period?

Reply:
Yes. It's also called a period.